package.json

```diff
@@ -233,10 +233,12 @@
     "webpack": "^5.70.0"
   },
   "resolutions": {
-    "@emotion/core": "10.0.28",
     "@storybook/react/webpack": "^5",
     "react-hot-loader": "^4.13.0",
-    "csstype": "^3.0.11"
+    "csstype": "^3.0.11",
+    "@emotion/core": "npm:@emotion/react@11.9.0",
+    "@emotion/css": "11.9.0",
+    "@storybook/theming/emotion-theming": "npm:@emotion/react@11.9.0"
```

Comment:
question: Why do you have mixed resolution syntaxes? I'm still not 100% sure I understand why we would need to explicitly say "resolve from npm" for some dependecies, when we only use one registry, which is npm.

Reply:
We addressed this offline, here is a summary:
- force resolution to different npm package and version
- temp fix validated, will change when we upgrade storybook to latest